adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -155,6 +183,7 @@ trait AkkaHttpAdapter {
         }
 
         val flow = Flow.fromSinkAndSource(sink, source).watchTermination() { (_, f) =>
+          //TODO: Shouldn't it send GQL_COMPLETE here before terminating?
```

Comment:
Hmm not here, but indeed the spec says that for subscriptions, GQL_COMPLETE should be sent when unsubscribing and this seems missing so far.

Reply:
I thought there might be better: https://github.com/ghostdogpr/caliban/blob/master/adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala#L150 but maybe this is called too so it might be fine if it's called on unsubscribing.

Reply:
I'll leave it up to you, and at any rate, it should be a different PR

Reply:
Fair enough, I'll do it in another PR. You can remove the TODOs then.